Composer/packages/client/src/App.tsx

```diff
@@ -8,18 +8,23 @@ import { useRecoilValue } from 'recoil';
 import { Header } from './components/Header';
 import { Announcement } from './components/AppComponents/Announcement';
 import { MainContainer } from './components/AppComponents/MainContainer';
+import { userSettingsState } from './recoilModel';
+import { loadLocale } from './utils/fileUtil';
 import { dispatcherState } from './recoilModel/DispatcherWrapper';
 
 initializeIcons(undefined, { disableWarnings: true });
 
 export const App: React.FC = () => {
+  const { appLocale } = useRecoilValue(userSettingsState);
+  loadLocale(appLocale);
```

Comment:
Should this be moved into a `useEffect`?

Reply:
Good idea. I'll fix that.